homeassistant/components/media_player/kodi.py

```diff
@@ -249,9 +286,6 @@ def __init__(self, hass, name, host, port, tcp_port, encryption=False,
             self._ws_server.Player.OnStop = self.async_on_stop
             self._ws_server.Application.OnVolumeChanged = \
                 self.async_on_volume_changed
-            self._ws_server.System.OnQuit = self.async_on_quit
```

Comment:
Don't get rid of these handlers. They're still used to track the state of Kodi. This is probably part of the second PR.